projects/packages/forms/src/blocks/contact-form/components/jetpack-integrations-modal/index.tsx

```diff
@@ -39,10 +38,14 @@ const IntegrationsModal = ( {
 		mailpoet: false,
 	} );
 
+	const config = useFormsConfig();
+
 	if ( ! isOpen ) {
 		return null;
 	}
 
+	const isMailPoetEnabled = Boolean( config?.isMailPoetEnabled );
+
```

Comment:
An example of replacing jpFormsBlocks method with the new getScriptsData() single-source-of-data method.